evaluator/builtin_time_test.go

```diff
@@ -316,6 +316,14 @@ func (s *testEvaluatorSuite) TestCurrentTime(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testEvaluatorSuite) TestUTCDate(c *C) {
```

Comment:
Can you add another case like:
mysql> select utc_date + 1;
+--------------+
| utc_date + 1 |
+--------------+
|     20160402 |
+--------------+